hoomd/hpmc/UpdaterRemoveDrift.h

```diff
@@ -43,10 +42,49 @@ template<class Shape> class RemoveDriftUpdater : public Updater
     public:
     //! Constructor
     RemoveDriftUpdater(std::shared_ptr<SystemDefinition> sysdef,
-                       std::shared_ptr<ExternalFieldLattice<Shape>> externalLattice,
-                       std::shared_ptr<IntegratorHPMCMono<Shape>> mc)
-        : Updater(sysdef), m_externalLattice(externalLattice), m_mc(mc)
+                       std::shared_ptr<IntegratorHPMCMono<Shape>> mc,
+                       pybind11::array_t<double> ref_positions)
+        : Updater(sysdef), m_mc(mc)
         {
+        setReferencePositions(ref_positions);
+        }
+
+    //! Set reference positions from a Nx3 numpy array
+    void setReferencePositions(const pybind11::array_t<double> ref_pos)
+        {
+        const size_t N = ref_pos.request().shape[0];
+        const size_t dim = ref_pos.request().shape[1];
+        if (N != this->m_pdata->getN() || dim != 3)
```

Comment:
@b-butler I believe we need `getNGlobal()` here or am I missing something?

Reply:
Likely so, unless we want to deal with particle migration.